woocommerce/compatibility/class-sv-wc-order-compatibility.php

```diff
@@ -522,6 +522,117 @@ public static function get_edit_order_url( \WC_Order $order ) {
 	}
 
 
+	/**
+	 * Gets the order meta according to HPOS availability.
+	 *
+	 * Uses WC_Order data store object if HPOS is enabled, otherwise it uses the traditional get_post_meta() method.
+	 *
+	 * @since x.y.z
+	 *
+	 * @param int $order_id Required. Order ID.
+	 * @param string $key Required. Meta key.
+	 * @param bool $single return first found meta with key, or all with $key.
+	 *             Defaults to true if HPOS is enabled, otherwise false.
+	 *
+	 * @return mixed
+	 */
+	public static function get_order_meta( int $order_id, string $key, $single = null) {
+
+		$is_hpos_enabled = SV_WC_Plugin_Compatibility::is_hpos_enabled();
+		$value = false;
+
+		// setting $single to false by default if hpos is disabled to match it with get_post_meta method's default value
```

Comment:
Is this comment entirely accurate? It says "setting $single to false by default if hpos is disabled" but won't this also set $single to true if HPOS is _enabled_? Is that intended?

Reply:
@jstern-godaddy please take a look at this above comment thread [here](https://github.com/godaddy-wordpress/wc-plugin-framework/pull/581#discussion_r1065946813) and let me know if I should just set it to false by default in our new method? I did this to avoid any issues while replacing in the plugins.